src/setup-maintain/setup/network-requirements.md

```diff
@@ -26,6 +26,10 @@ The OutSystems services use the following ports:
 
 * 12002 - OutSystems Scheduler Service
 
+> **Security Tips**: Those ports shouldn't be open to the internet.
```

Comment:
Consider removing the "Open Ports" section, as that information is already further down in subsequent tables.

The "Security Tip" is indeed helpful, but could be also "nearer" to the same tables - as it applies to all the ports mentioned there.